scripts/release.sh

```diff
@@ -95,6 +150,7 @@ RELEASE_BRANCH=""
 RELEASE_GCS_BUCKET="knative-nightly/${REPO_NAME}"
 KO_FLAGS="-P"
 VALIDATION_TESTS="./test/presubmit-tests.sh"
```

Comment:
This is currently immutable, you might want to add a flag to use a different validation test. Up to you.

Reply:
Hmm - good suggestion... will do!